test/tt_store_suite.erl

```diff
@@ -0,0 +1,32 @@
+-module(tt_store_suite).
```

Comment:
We have Commont Test test suite: AWESOME! 

However, its name should be `tt_store_SUITE.erl` - the convention is that the `SUITE` is written in capitals and rebar understands that (you can run the suites then with `./rebar3 ct`).